pkg/sqlite/metrics.go

```diff
@@ -185,3 +188,19 @@ func ReadMetrics(gatherer prometheus.Gatherer) (Metrics, error) {
 
 	return mtr, nil
 }
+
+// Computes the QPS for insert/updates, deletes, and selects, based on the previous and current metrics time.
+func (prev Metrics) QPS(cur Metrics) (insertUpdateAvgQPS float64, deleteAvgQPS float64, selectAvgQPS float64) {
+	insertUpdateAvgQPS = float64(0)
+	deleteAvgQPS = float64(0)
+	selectAvgQPS = float64(0)
+
+	if !prev.IsZero() && !cur.IsZero() {
+		elapsedSeconds := cur.Time.Sub(prev.Time).Seconds()
```

Comment:
need to handle elapsedSeconds zero case, incase divide by zero happens, rest lgtm

Reply:
@cardyok fixed, thx